purchase_operating_unit/security/purchase_security.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--Copyright 2015-17 ForgeFlow S.L.-->
+<!-- - Jordi Ballester Alomar-->
+<!--Copyright 2015-17 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya-->
+<!--License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).-->
+<odoo noupdate="1">
+    <record id="ir_rule_purchase_order_allowed_operating_units" model="ir.rule">
+        <field name="model_id" ref="purchase.model_purchase_order" />
+        <field
+            name="domain_force"
+        >['|',('operating_unit_id','=',False),('operating_unit_id','in', user.operating_unit_ids.ids)]</field>
```

Comment:
@jdidderen-nsi I think you should double-check this place using this link https://github.com/OCA/operating-unit/pull/667#discussion_r1629838296

Reply:
Fixed 👍